com.unity.cinemachine/Samples~/3D Samples.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
```

Comment:
There shouldn't be metafiles for the folders in this directory.  If they're here, it's a mistake.  You shouldn't be committing new ones.

Reply:
Fixed